keyboards/cyboard/dactyl/info.json

```diff
@@ -0,0 +1,744 @@
+{
+    "keyboard_name": "Dactyl",
+    "usb": {
+        "pid": "0x0000",
+        "device_version": "0.0.1"
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "rows": ["GP8", "GP7", "GP28", "GP6", "GP23", "GP27", "GP21"],
+        "cols": ["GP9", "GP26", "GP22", "GP20", "GP29", "GP4", "GP5"]
+    },
+    "split": {
+        "soft_serial_pin": "GP0",
+        "soft_serial_speed": 2
```

Comment:
Is there a reason for this?

Reply:
The slave side RGB would flicker or cut out entirely once every 5-30min.  Slowing down the serial speed fixed the issue.  This is with splinky v3 controllers, btw.  I'm testing the USART half duplex to see if that is a cleaner fix.

Reply:
Thankfully usart [baudrate](https://docs.qmk.fm/#/serial_driver?id=baudrate) can be adjusted 🙂